src/utils/dom/querySelectorAll.js

```diff
@@ -12,5 +12,22 @@ governing permissions and limitations under the License.
 
 import toArray from "../toArray";
 
-export default (context, selector) =>
-  toArray(context.querySelectorAll(selector));
+const SIBLING_PATTERN = /^\s*>/;
+
+export default (context, selector) => {
+  if (!SIBLING_PATTERN.test(selector)) {
+    return toArray(context.querySelectorAll(selector));
+  }
+
+  const tag = `alloy${+new Date()}`;
+
+  try { // adding a dummy css class to be able to select the children
+    context.classList.add(tag); // needs to be IE compliant
```

Comment:
I'm not sure what you mean by this. From what I can tell classList.add/remove is supported in IE 10 and above.

Reply:
@jonsnyder it was a placeholder for me to double check IE compatibility. According to MDN it is supported in IE 10+ and modern browsers. I have removed the comments.